source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -215,10 +215,15 @@ def expand(self,unit):
 				self.setEndPoint(docInfo,"endToEnd")
 
 	def getTextWithFields(self,formatConfig=None):
-		if self.isCollapsed:
-			# #7652: We cannot fetch fields on collapsed ranges otherwise we end up with repeating controlFields in braille (such as list list list). 
-			return []
-		fields=super(WordDocumentTextInfo,self).getTextWithFields(formatConfig=formatConfig)
+		fields = None
+		if not self.isCollapsed:
+			rawText = self._rangeObj.GetText(2)
+			if not rawText or rawText == '\x07':

```

Comment:
What does `'\x07'` represent in this instance? Could it be replaced by a named constant?